samples/Kconfig

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2018 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+config NCS_SAMPLES_DEFAULTS
```

Comment:
How will customers relate to this config? What happens if they modify it in menuconfig?  Are they meant to do that? Maybe we should take steps to discourage that if it is not desired? E.g. remove prompt, put a warning in the prompt/help text etc.

Reply:
Nothing should happen. It would work only when selected during project generation.

Reply:
Right, that makes sense. It should probably be explained in the help text, so they don't think that it is a DEBUG toggle switch or something.